pkg/ctl/generate/profile.go

```diff
@@ -0,0 +1,84 @@
+package generate
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/git"
+	"github.com/weaveworks/eksctl/pkg/gitops"
+	"github.com/weaveworks/eksctl/pkg/gitops/fileprocessor"
+)
+
+const (
+	defaultGitTimeout = 20 * time.Second
+)
+
+// Command creates `generate` commands
+func Command(flagGrouping *cmdutils.FlagGrouping) *cobra.Command {
+	verbCmd := cmdutils.NewVerbCmd("generate", "Generate GitOps manifests", "")
+	cmdutils.AddResourceCmd(flagGrouping, verbCmd, generateProfileCmd)
+	return verbCmd
+}
+
+type options struct {
+	gitops.GitOptions
+	ProfilePath string
+}
+
+func generateProfileCmd(rc *cmdutils.Cmd) {
+	cfg := api.NewClusterConfig()
+	rc.ClusterConfig = cfg
+
+	rc.SetDescription("profile", "Generate a GitOps profile", "")
+
+	var o options
+
+	rc.SetRunFuncWithNameArg(func() error {
+		return doGenerateProfile(rc, o)
+	})
+
+	rc.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+		fs.StringVarP(&o.URL, "git-url", "", "", "Git repository URL")
+		fs.StringVarP(&o.Branch, "git-branch", "", "master", "Git branch; defaults to master")
+		fs.StringVarP(&o.ProfilePath, "profile-path", "", "", "Path to generate the profile in; defaults to CWD")
```

Comment:
Shouldn't the default be `.` and avoid saying it explicitly?

Reply:
If so, I'd rather see it as './', a dot can be hard to notice.